features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/utils/DefaultColorStringFormatter.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.utils
+
+import com.datadog.android.core.internal.utils.HEX_RADIX
+import okhttp3.internal.and
```

Comment:
we shouldn't use `okhttp3.internal` package, it can be removed/changed at any point